src/components/UI/Buttons/Buttons.tsx

```diff
@@ -1,19 +1,66 @@
 import { JSX } from 'preact';
 import styles from './Buttons.module.sass';
+import backIcon from './icons/backArrow.svg';
+import nextIcon from './icons/nextArrow.svg';
 
 interface ButtonProps extends JSX.HTMLAttributes<HTMLButtonElement> {
-  onClick: () => void;
-  children: JSX.Element | string;
+  onClick: (() => void) | undefined;
+  children: JSX.Element | JSX.Element[] | string;
```

Comment:
No need for `onClick` and `children` because it's part of `JSX.HTMLAttributes<HTMLButtonElement>`.
```suggestion
```